CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# March 18, 2019
+### Rely 1.3.0
+* jUnit reporter
+* throw error when nesting top level TestFramework.describe
+* Fix bug expect.string.toMatch where would only check matches beginning on first character,
```

Comment:
I would make this into a few bullets:

- Fix bug in `expect.string.toMatch` where it would only check matches beginning on the first character (Now aligned with Jest's behavior)
- Switch internal implementation from [Str](http://caml.inria.fr/pub/docs/manual-ocaml/libref/Str.html) to [Re](https://github.com/ocaml/ocaml-re)